oasis/models/iforest.py

```diff
@@ -132,55 +124,43 @@ def run(self):
         for key in self.job.metrics:
             if key in Metrics:
                 val = Metrics[key]
-                config = {"features": ["mean", "std"]}
-                if key in self.cfg.metrics and "features" in self.cfg.metrics[key]:
-                    config = self.cfg.metrics[key]
+                if key not in self.cfg.metrics:
+                    continue
 
-                t = Thread(target=self.run_action, args=(key, val, config, ))
+                t = Thread(target=self.run_action,
+                           args=(key, val, self.cfg.metrics[key], ))
```

Comment:
Remove the last comma, it is only needed if this tuple contains only one element